packages/vehicle_detection/config/vehicle_filter_node/default.yaml

```diff
@@ -1,3 +1,3 @@
 distance_between_centers: 0.0125
 max_reproj_pixelerror_pose_estimation: 1.5
-virtual_stop_line_offset: 0.15
\ No newline at end of file
+virtual_stop_line_offset: 0.5
```

Comment:
This is to trigger the vehicle stop distance as "safe" as possible. Due to FPS drop and lag, sometimes frames won't come in fast enough for a safe stop. This way, it is guaranteed that when the distance is 0.5 away, it will stop the vehicle, assuming when you see 0.5, the vehicle has also driven forward more at about 1 second at speed of 0.3. 

This should really be addressed once we have a more "realtime" detection system.